tdms/include/mesh_base.h

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file mesh_base.h
+ * @brief TODO: what are these functions
```

Comment:
I think these are the functions that are meant to allow for extraction of the field over a given surface. The user can specify (EG a plane) to extract the field on, and these are the functions which are meant to mesh it.
Don't quote me on that though!

Reply:
Do be fair, some documentation is in the source and gets moved/tidied/improved as part of Tom's refactoring. So we can just leave the TODO for now (to avoid annoying him with conflicts).

```
Generate an oriented mesh on the surface of a cuboid
within the FDTD grid.
 ```